packages/flutter/test/material/text_selection_toolbar_text_button_test.dart

```diff
@@ -123,5 +123,105 @@ void main() {
         customForegroundColor,
       );
     });
+
+    testWidgetsWithLeakTracking('background color by default', (WidgetTester tester) async {
```

Comment:
Are we supposed to use testWidgetsWithLeakTracking for everything now?

Reply:
Not sure about that. @polina-c do you have any thoughts on when we should use `testWidgetsWithLeakTracing`?